src/routes/capacity.js

```diff
@@ -16,7 +21,6 @@ const formatCapacityResult = capacity => ({
     asset,
   })),
   allocatedNxm: capacity.usedCapacity.toString(),
-  utilizationRate: capacity.utilizationRate.toNumber(),
```

Comment:
this is the structure change. move `utilizationRate` from product level to pool level